packages/block-library/src/button/edit.js

```diff
@@ -201,7 +201,7 @@ function ButtonEdit( props ) {
 	return (
 		<>
 			<ColorEdit { ...props } />
-			<Block.div>
+			<div { ...useBlockWrapperProps() }>
```

Comment:
I think we should set a guideline that we'd avoid to use this inline to avoid the React hooks order issue even if the component doesn't have early returns (they can be added inadvertently later).

Reply:
When would there be an order issue when there is not early returns?

Reply:
But I'm fine with avoiding inline uses. I don't have a strong opinion.

Reply:
> When would there be an order issue when there is not early returns?

never, it's just that someone editing the block in the future, might not even notice that there's a hook in the returned function. It's not common to use hooks in the returned element in React components.

Reply:
Sure, I'll adjust it :)

Reply:
Done